ches/emailHandler.ts

```diff
@@ -15,25 +18,59 @@ export const sendCreateRequestEmails = async (formData: PrivateCloudCreateReques
         .filter(Boolean)
         .map((item) => item?.email),
       from: 'Registry <PlatformServicesTeam@gov.bc.ca>',
-      subject: `${formData.name} deletion request received`,
+      subject: `${formData.name} provisioning request received`,
     });
-    // await chesService.send({
-    //   bodyType: "html",
-    //   body: email,
-    //   to: adminEmails,
-    //   from: "Registry <PlatformServicesTeam@gov.bc.ca>",
-    //   subject: `New Delete request in Registry waiting for your approval`,
-    // });
+    await chesService.send({
+      bodyType: 'html',
+      body: email,
+      to: adminEmails,
+      from: 'Registry <PlatformServicesTeam@gov.bc.ca>',
+      subject: `New Provisioning request in Registry waiting for your approval`,
+    });
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+export const sendRequestApprovalEmails = async (request: PrivateCloudRequestWithRequestedProject) => {
+  const email = render(RequestApproval({ request }), { pretty: true });
+  try {
+    await chesService.send({
+      bodyType: 'html',
```

Comment:
I understand that the majority of templates are in `html` format. Would it be possible to set html as the default `bodyType` value in the `ches service`, allowing for the option to override it when calling the send function?

Reply:
Since we are creating each function for specific requests/emails, I don't think that will be necessary since we will create a separate function anyways if we need to send a different bodyType.

Reply:
What I'm suggesting is to establish a default value for the `bodyType` option within the send function. Default values are used when specific options are not provided, making them useful in most cases.
I see `email: any` in `send` function; it would be nicer to define the email type there as well.